### PR TITLE
Implemented functional looms

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: server/session/handler_loom.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	loomInputSlot = 0x09
+	// loomDyeSlot is the slot index of the dye item in the loom table.
+	loomDyeSlot = 0x0A
+	// loomPatternSlot is the slot index of the pattern item in the loom table.
+	loomPatternSlot = 0x0B
+)
+
+// handleLoomCraft handles a CraftLoomRecipe stack request action made using a loom table.
+func (h *ItemStackRequestHandler) handleLoomCraft(a *protocol.CraftLoomRecipeStackRequestAction, s *Session) error {
+	// First check if there actually is a loom opened.
```
Comment (TwistedAsylumMC) [inline server/session/handler_loom.go:21]: Maybe some more documentation? You say "First ...." but have no other comments

Comment (JustTalDevelops) [reply server/session/handler_loom.go:21]: Added.

**Review by DaPigGuy — APPROVED** [review]

**Review by T14Raptor — APPROVED** [review]

**Review by TwistedAsylumMC — APPROVED** [review]

File: server/session/handler_loom.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+const (
+	// loomInputSlot is the slot index of the input item in the loom table.
+	loomInputSlot = 0x09
+	// loomDyeSlot is the slot index of the dye item in the loom table.
+	loomDyeSlot = 0x0A
+	// loomPatternSlot is the slot index of the pattern item in the loom table.
+	loomPatternSlot = 0x0B
```
Comment (Sandertv) [inline server/session/handler_loom.go:16]: Would make this hex lowercase for consistency.

**Review by Sandertv — CHANGES_REQUESTED** [review]
One small comment, looks good otherwise.

**Review by Sandertv — APPROVED** [review]
This looks good too. Thanks for the PR!

**Pull request merged**